dotnetv3/AutoScaling/Tests/AutoScalingWrapperTests.cs

```diff
@@ -0,0 +1,321 @@
+        /// </summary>
+        /// <returns>Async Task.</returns>
+        [Fact()]
+        [Order(6)]
+        [Trait("Category", "Integration")]
+        public async Task SetDesiredCapacityAsyncTest()
+        {
+            var newMax = 3;
+            var success = await _autoScalingWrapper.SetDesiredCapacityAsync(_groupName, newMax);
+            Assert.True(success, "Couldn't set the desired capacity.");
+            _groups = await _autoScalingWrapper.DescribeAutoScalingGroupsAsync(_groupName);
+            _groups.ForEach(group =>
+            {
+                if (group.AutoScalingGroupName == _groupName)
+                {
+                    Assert.Equal(newMax, group.DesiredCapacity);
+                }
+            });
+        }
+
+        /// <summary>
+        /// Test the UpdateAutoScalingGroup method by changing the maximum
+        /// capacity. Once the change is made, the DescribeAutoScalingGroupsAsync
+        /// method checks the capacity for the group.
+        /// </summary>
+        /// <returns>Async Task.</returns>
+        [Fact()]
+        [Order(7)]
+        [Trait("Category", "Integration")]
+        public async Task UpdateAutoScalingGroupTest()
+        {
+            var newMax = 3;
+            var success = await _autoScalingWrapper.UpdateAutoScalingGroupAsync(_groupName, _launchTemplateName, _serviceLinkedRoleArn, newMax);
+            Assert.True(success, $"Couldn't update the Auto Scaling group: {_groupName}.");
+            _groups = await _autoScalingWrapper.DescribeAutoScalingGroupsAsync(_groupName);
+            _groups.ForEach(group =>
+            {
+                if (group.AutoScalingGroupName == _groupName)
+                {
+                    Assert.Equal(newMax, group.DesiredCapacity);
+                }
+            });
+        }
+
+        /// <summary>
+        /// Test the DescribeAutoScalingGroupsAsync method.
+        /// </summary>
+        /// <returns>Async Task.</returns>
+        [Fact()]
+        [Order(8)]
+        [Trait("Category", "Integration")]
+        public async Task DescribeAutoScalingGroupsAsyncTest()
+        {
+            var details = await _autoScalingWrapper.DescribeAutoScalingGroupsAsync(_groupName);
+            Assert.True(details.Count > 0, $"Couldn't find that Auto Scaling group {_groupName}.");
+        }
+
+        /// <summary>
+        /// Test the DescribeAutoScalingActivitiesAsync method. The method
+        /// should return a non-null collection of activities.
+        /// </summary>
+        /// <returns>Async Task.</returns>
+        [Fact()]
+        [Order(9)]
+        [Trait("Category", "Integration")]
+        public async Task DescribeScalingActivitiesAsyncTest()
+        {
+            var activities = await _autoScalingWrapper.DescribeScalingActivitiesAsync(_groupName);
+            Assert.NotNull(activities);
+            //Assert.True(activities.Count > 0, $"Can't find any auto scaling activities for the group: {_groupName}.");
```

Comment:
```suggestion
            //Assert.True(activities.Count > 0, $"Can't find any Amazon EC2 Auto Scaling activities for the group: {_groupName}.");
```

Reply:
That was an earlier iteration of the test and the commented code has been removed.